plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -151,7 +151,7 @@ public String getDescription() {
     @NonNull
     @Override
     public Permission getRequiredPermission() {
-        return Jenkins.SYSTEM_READ;
+        return Jenkins.READ;
```

Comment:
Is this wise? Maybe you could explain once more why this needs to be READ?

Reply:
This is only for showing the link on manage, it doesn’t reveal any info or allow access to any data.

It’s a workaround used in a couple of places for where both system read and manage need access to a page because management link only allows a single permission to govern access